### PR TITLE
Moved newsletters to root of route folder

### LINKED ISSUE #2232: 💸 Newsletters pages under /images/ folder?!
cc @bradystroud @BrookJeynes @wicksipedia 

**Description**  

Hi Team,

looking at a Google Tag Manager report from @camillars I noticed this bunch of links under `https://www.ssw.com.au/images/newsletter-uploads/...`

For a moment I thought it was a bug, but navigating from https://www.ssw.com.au/newsletters I confirmed the newsletters pages are actually under an /images folder...

**Proposed solution**  

- [x] 1. Move the files from `https://www.ssw.com.au/images/newsletter-uploads/YYYY/...` to `https://www.ssw.com.au/newsletters/YYYY/...`
- [ ] 2. Add temporary redirects so it doesn't affect Google juice. Maybe remove the redirects after 1 year, when the new path will be crawled already

**Screenshots**
Screenshot 2024-03-04 at 12 13 09 PM

Screenshot 2024-03-04 at 12 13 19 PM

### PR DESCRIPTION
**Pull request by Calinator444**
##### Changes 

- Fixed broken meetup logo throughout newsletters
- Fixed broken images of Adam at microsoft Roadshow
- Fixed broken SQLAuditor logo image links

#####  Affected routes: 
- images/newsletter-uploads/*
- newsletter-uploads/* (new route)

#####  Acceptance Criteria
- [x] New newsletrers get uploaded to `/newsletter-uploads` and not `/images/newsletter-uploads`
- [x] Existing Tina articles with images display properly
- [x] Redirects for old newsletter path set up 
- [x] All image files upload to the correct location

##### Important Note

- I've also modified the existing newsletter files in tina to link to the correct newsletters (e.g. `/newsletter-uploads/2024/_2024_03__.NET Migrations Made Easy__.html` instead of `images/newsletter-uploads/2024/_2024_03__.NET Migrations Made Easy__.html`
- in addition I've modified tina so that it will save newsletters to the correct location and not under `/images`

Fixes #2232 

- [ ] If adding a new page, I have followed the [📃 New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) issue template

- [x] Include done video or screenshots

##### Screenshots

image

**Figure**: **Bad - newsletter stored under /imagesa/newsletter-uploads**

image
**Figure**: **Good - newsletter stored under /newsletter-uploads on staging**

nextjs_redirects
**Figure**: **Using Next.js to redirect traffic from old newsletter path to new newlsetter path**

### CODE REVIEW

File: components/blocks/customImage.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import classNames from "classnames";
 import Image from "next/image";
-import type { Template } from "tinacms";
+import type { Collection, Template } from "tinacms";
```
Comment (wicksipedia) [inline components/blocks/customImage.tsx]:
```suggestion
import type { Template } from "tinacms";
```
as per linter

File: middleware.ts [hunk 2]
```diff
@@ -1,10 +1,15 @@
+import { NextURL } from "next/dist/server/web/next-url";
 import { NextRequest, NextResponse } from "next/server";
 import { addNoIndexHeaders } from "./middleware/noIndex";
 
 export function middleware(request: NextRequest) {
   const response = NextResponse.next();
-
+  const url: NextURL = request.nextUrl.clone();
+  let path = url.pathname.toString();
```
Comment (wicksipedia) [inline middleware.ts]:
```suggestion
  const path = url.pathname.toString();
```
as per linter

**Review by wicksipedia — CHANGES_REQUESTED** [review]

**Comment by wicksipedia** [comment]
looks like your latest code isn't being built/deployed due to linting errors

**Comment by Calinator444** [comment]
@wicksipedia closing this because merging would break all of the images on the site"

I'll hang onto the branch because it will be useful when we integrate a media manager for tina later and we can fix all of the file paths

**Pull request closed without merging**